content/mooreslaw-tutorial.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<p>If you want to share data <strong>(and?)</strong> view the results in a table - something is missing in this sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>I missed this, current form doesn't make sense. I wanted simplify the use case here e.g. You want to view results in a table and share a table with other users. </p><p> </p><p>"share data <strong>and</strong> view the results" makes sense.</p>